wordpress-mobile/WordPress-iOS
# Corrected confirmation text displayed after clearing shortcut suggestions

Fixes #10553

To test:

1. Checkout the branch and confirm that test pass.
1. From Me : App Settings, tap `Clear Siri Shortcut Suggestions`.
1. Verify that the notice displayed after tapping it matches the screenshot below. In English, this now reads `Successfully cleared Siri Shortcut Suggestions`.

<img width="545" alt="10553" src="https://user-images.githubusercontent.com/221062/49223285-a0270080-f3de-11e8-8c14-052659ce577e.png">
